reconcile/utils/ocm.py

```diff
@@ -414,6 +442,7 @@ def _get_create_cluster_spec(cluster_name: str, cluster: OCMSpec) -> dict[str, A
                     "rosa_creator_arn": cluster.spec.account.rosa.creator_role_arn
                 },
                 "product": {"id": "rosa"},
+                "flavour": {"id": "osd-4"},
```

Comment:
Can you elaborate on this?

Reply:
Flavours are used to define clusters' infrastructure options such as instance types, disk sizes, etc. 
This value is used by ROSA CLI and it's used in the payload examples I got from the OCM team. Its name is  "Openshift Dedicated 4.x" and is the default one.  

I've tried to remove it and ClustersService uses it as default, so I think it's better not to set the attribute and let CS set the default value here.

Reply:
Thank you!